iec61499_to_svg.py

```
@@ -1298,11 +1316,11 @@ def _render_external_labels(self, fb: FunctionBlock) -> str:
             if self.show_types or (self.show_comments and port.comment):
                 label_parts = []
                 if self.show_comments and port.comment:
-                    label_parts.append(port.comment)
+                    label_parts.append(_xml_escape(port.comment))
                 if self.show_types:
                     if label_parts:
                         label_parts.append(" – ")
```

Ruff (0.16.3)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)